### PR TITLE
Add GriefDefender support.

### PR DESCRIPTION
**Pull request by bloodmc**
## Additions
- Adds protection support for [GriefDefender](https://www.spigotmc.org/resources/68900/).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +47,11 @@
       <id>placeholderapi</id>
       <url>https://repo.extendedclip.com/content/repositories/placeholderapi/</url>
     </repository>
+    <!-- GriefDefender API Repo -->
+    <repository>
+	  <id>glaremasters</id>
```
Comment (Aztlon) [inline pom.xml:52]: Could you indent it the same as the other repositories? Same with the dependency below.

**Review by Aztlon — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (bloodmc) [reply pom.xml]: Should be good now.

**Review by Aztlon — APPROVED** [review]
Thanks, looks good!

**Review by ChristopherWMM — APPROVED** [review]
Thanks for the contribution, really appreciate you doing this! Looks great! 👍

**Pull request merged**